tests/integration/features/bootstrap/FeatureContext.php

```diff
@@ -5302,4 +5333,55 @@ protected function assertStatusCode(ResponseInterface $response, int $statusCode
 			Assert::assertEquals($statusCode, $response->getStatusCode(), $message);
 		}
 	}
+
+	/**
+	 * @param string $user
+	 * @param string $identifier
+	 * @param int $statusCode
+	 * @param string $apiVersion
+	 * @param TableNode|null $formData
+	 * @return void
+	 *
+	 * @Given /^user "([^"]*)" creates conversation with event "([^"]*)" \((v4)\)$/
+	 */
+	public function createCalendarEventConversation(string $user, string $identifier, string $apiVersion = 'v1', ?TableNode $formData = null): void {
+		$body = $formData->getRowsHash();
+		if (isset($body['objectId'])) {
+			[$start, $end ] = explode('#', $body['objectId']);
+			$startTime = time() + (int)$start;
+			$endTime = time() + (int)$end;
+			$body['objectId'] = ($startTime) . '#' . ($endTime);
+			self::$identifierToObjectId[$identifier] = $body['objectId'];
+		}
+
+		$this->setCurrentUser($user);
+		$this->sendRequest('POST', '/apps/spreed/api/' . $apiVersion . '/room', $body);
+		$this->assertStatusCode($this->response, 201);
+
+		$response = $this->getDataFromResponse($this->response);
+
+		self::$identifierToToken[$identifier] = $response['token'];
+		self::$identifierToId[$identifier] = $response['id'];
+		self::$tokenToIdentifier[$response['token']] = $identifier;
+
+		$location = self::getRoomLocationForToken($identifier);
+		foreach (['LOCAL'] as $server) {
```

Comment:
should not do this loop and just use the currently configured server, so we can test federation in the future the same way.